app/components/Intercept_Components/InterceptTextBox.tsx

```diff
@@ -1,33 +1,30 @@
 import * as React from "react";
 import { RequestHeaderList } from "./RequestHeaderList";
-import { responseField, statusCodes, contentType } from "./../../content/content";
+//import { responseField, statusCodes, contentType } from "./../../content/content";
 interface InterceptTextBox {
+  data: any;
   rowProps: any;
-  responseText: responseField;
-  statusCodes: statusCodes;
-  contentType: contentType;
-  responseError: responseField;
   fetchResponse: any;
-  responseData: responseField;
   handleRespTextChange: any;
   handleStatusCodeChange: any;
   handleContentTypeChange: any;
+  currentTabId: number;
 }
 
 export const InterceptTextBox: React.SFC<InterceptTextBox> = props => {
   const requestId = props.rowProps.checkbox.requestId;
   const defaultResponseText = "";
   const defaultStatusCode = "200";
   const defaultContentType = "application/json";
-  const responseTextValue = props.responseText[requestId] || defaultResponseText;
-  const statusCodeValue = props.statusCodes[requestId] || defaultStatusCode;
-  const contentTypeValue = props.contentType[requestId] || defaultContentType;
+  const responseTextValue = props.data.responseText[requestId] || defaultResponseText;
+  const statusCodeValue = props.data.statusCodes[requestId] || defaultStatusCode;
+  const contentTypeValue = props.data.contentType[requestId] || defaultContentType;
```

Comment:
All three default values should go in `initialTabProperties` in reducer.

Reply:
Since `requestId` and `tabId` are dynamic values that get saved in the store after dispatching an `INITIALISE_DEFAULTS` action , how do I add these in the reducer without the values of `tabId` or `requestId`